schemas/com.mailgun/message_clicked/jsonschema/1-0-0

```diff
@@ -0,0 +1,75 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for a Mailgun message clicked event",
+	"self": {
+		"vendor": "com.mailgun",
+		"name": "message_clicked",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+
+	"type": "object",
+	"properties": {
+		"recipient": {
+			"type": "string"
+		},
+		"city": {
+			"type": "string"
+		},
+		"ip": {
+			"type": "string",
+			"format": "ipv4"
+		},
+		"body-plain": {
+			"type": "string"
+		},
+		"timestamp": {
+			"type": "string"
+		},
+		"url": {
+			"type": "string",
+			"format": "uri"
+		},
+		"event": {
+			"type": "string"
+		},
+		"user-agent": {
+			"type": "string"
+		},
+		"domain": {
+			"type": "string"
+		},
+		"signature": {
+			"type": "string",
+			"pattern": "^([A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{4}|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{2}==)$"
+		},
+		"country": {
+			"type": "string"
+		},
+		"client-type": {
+			"type": "string"
+		},
+		"client-os": {
+			"type": "string"
+		},
+		"token": {
+			"type": "string"
+		},
+		"client-name": {
+			"type": "string"
+		},
+		"region": {
+			"type": "string"
+		},
+		"device-type": {
+			"type": "string"
+		},
+		"my_var_1": {
+			"type": "string"
+		},
+		"my-var-2": {
+			"type": "string"
+		}
```

Comment:
same here

Reply:
Ok I found out where these come from. When you create a webhook with mailgun you are given the option to test it and see the result in the postbin. Unfortunately testing it creates some additional variables (my_var_1, and  my-var-2). Furthermore it does not contain some fields (h and message-id), and some of the documented optional fields (campaign-id, campaign-name, mailing-list, tag). I am going through those schemas again now.

Reply:
Hey @knservis - thanks for digging into this. So just to confirm, those my_var_1, and my-var-2 are Mailgun-authored fields (albeit test path only) - no operator involvement?

Reply:
@alexanderdean that is correct for the test, although these extra variables could be anything (e.g. var-your-42-with-salt) as far as I can tell and that depends on the operator. It seems that the best way to handle those in the iglu schema would be with: 
```
 "additionalProperties": true
```

Reply:
The same should also happen for the "attachment-x" field which in fact translates to "attachment-1", "attachment-2" et c.

Reply:
Thanks. What does "that depends on the operator" mean? What's an operator?

Reply:
@alexanderdean yes

Reply:
@alexanderdean *operator = integrator

Reply:
Thanks, in that case I think we have to remove the my-vars and attachment-xs, and set additionalProperties: true...